tests/shared.ts

```diff
@@ -38,3 +38,11 @@ export async function clickMap(page: Page, x: number, y: number) {
     position: { x, y },
   });
 }
+
+export async function getLocalStorage(page: Page, key: string): Promise<any> {
+  let string = await page.evaluate(
```

Comment:
https://playwright.dev/docs/evaluating
This seems like the most straightforward way to get local storage